package utils

func FailOnError(err error, msg string) {
	
}